packages/jsii/bin/jsii.ts

```diff
@@ -38,11 +46,20 @@ import { VERSION } from '../lib/version';
 
   const projectInfo = await loadProjectInfo(projectRoot, { fixPeerDependencies: argv['fix-peer-dependencies'] });
 
+  // disable all silenced warnings
+  for (const key of argv['silence-warnings']) {
+    if (!(key in enabledWarnings)) {
+      throw new Error(`Unknown warning type ${key}. Must be one of: ${warningTypes}`);
+    }
+
+    (enabledWarnings as any)[key] = false;
```

Comment:
This `as any` shouldn't be necessary (see suggestion later in the review).

Reply:
Fixed